src/Project/countryDetails.jsx

```diff
@@ -0,0 +1,92 @@
+          </Link>
+          <div>
+            <div className="sm:p-2 md:p-3 sm:mt-2 md:m-2 md:ml-[40vw] lg:mt-[1vh] sm:ml-6">
+              {wait()}
+            </div>
+            {countries.map((country, index) => (
+              <div>
+              <div key={index} className="sm:p-2 md:p-3 sm:mt-2 md:m-2 md:ml-[20vw] lg:ml-[30vw] sm:ml-6 sm:w-[80vw] md:w-[50vw] lg:w-[20vw]">
```

Comment:
Using index as key can cause problem. Only use index as key if there is no unique key in the array itself. For this case, use a unique value from the countries array.